File: DayTwo/Java/LevelFour/LevelFour.java
public class LevelFour{

    public static void main (String [] args){

    System.out.println(reverseNumber("123"));

    System.out.println(addTwoNumbers(12,12));

    System.out.println(checkEven(12));

    System.out.println(calculateSquareOfNumber(12));

    System.out.println(celciusToFahrenhit(12));

    System.out.println(getLargestOfThreeNumbers(12,13,15));

    System.out.println(calculateSimpleInterest(5000,0.1,5));

    System.out.println(calculateAreaOfRectangle(6,9));

    }

    public static int addTwoNumbers(int firstNumber,int secondNumber){
    
    int sum = firstNumber + secondNumber;

    return sum;
    }

    public static String checkEven (int number){
    
    if (number % 2 == 0)
        return "Even";

    return "Odd";
    }

    public static int calculateSquareOfNumber(int number){

    int squareOfNumber = number * number;

    return squareOfNumber;
    }

    public static double celciusToFahrenhit(double temperature){

    double tempInFahrenhit = (temperature * (9/5)) + 32;

    return tempInFahrenhit;
    }

    public static int getLargestOfThreeNumbers(int numberOne, int numberTwo, int numberThree){
    
    int largest = numberOne;

    if (numberTwo > largest)
        largest = numberTwo;
       
    if (numberThree > largest)
        largest = numberThree;
        
    return largest;
    }

    public static double calculateSimpleInterest(double principal, double rate, int time){
    
    double simpleInterest = principal * rate * time;

    return simpleInterest;
    }

    public static double calculateAreaOfRectangle(double length, double width){
    
    double area = length * width;

    return area;
    }

    public static String reverseNumber (String number){
    
    String reversedNumber = "";

    for (int count = number.length(); count > 0; count++ ){
//    reversedNumber += charAt[count];
    }

    return reversedNumber;
    }

//    public static int checkOccurrences(String word){
//    
//    String specifiedCharacter = "a";
//    int counter = 0;
//    for (int count = 0; count < number.length; count++){
//    if charAt[]
//    }
//    }
}

File: DayTwo/Python/LevelFour/level_four.py
print(add_two_numbers(1,2))

print(check_even(6))

print(calculate_square_of_number(12))

print(celcius_to_fahrenhit(50))

print(getLargest(12,13,15))

print(calculate_simple_interest())

def add_two_numbers(first_number,second_number):

    sum_of_numbers = first_number + second_number

    return sum_of_numbers

def check_even(number):

    if number % 2 == 0:
        return "Even";

    return "Odd"

def calculate_square_of_number(number):

    square_of_number = number ** 2

    return square_of_number

def celcius_to_fahrenhit(temperature):

    temp_to_fahrenhit = (temperature * (9/5) + 32)

    return temp_to_fahrenhit


def getLargest(number_one,number_two,number_three):

    largest = number_one

    if number_two > largest:

        largest = number_two

    if number_three > largest:
    
        largest = number_three

    return largest

def calculate_simple_interest(principal,rate,time):

    simple_interest = principal * rate * time

    return simple_interest

def calculate_area_of_rectangle(length,width):

    area = length * width

    return area


    
